src/util/BaseLoginRouter.js

```diff
@@ -183,6 +185,10 @@ function (Okta, Backbone, xdomain, RefreshAuthStateController, Settings, Header,
       var oldController = this.controller;
       this.controller = new Controller(controllerOptions);
 
+      this.listenTo(this.controller, 'all', function () {
```

Comment:
nit: Looking at it now, we can do this (also note that we should add a comment describing why we're doing this):
```javascript
// Bubble up all controller events
this.listenTo(this.controller, 'all', this.trigger);
```